conference.js

```diff
@@ -44,7 +44,7 @@ import {
     onStartMutedPolicyChanged,
     p2pStatusChanged,
     sendLocalParticipant,
-    setDesktopSharingEnabled
+    setDesktopSharingEnabled, conferenceWillLeave, getCurrentConference
```

Comment:
Could you put` , conferenceWillLeave, getCurrentConference` on a new lines like the other imports?

Reply:
- [ ] separate imports